pallets/idn-manager/src/lib.rs

```diff
@@ -368,8 +394,10 @@ pub mod pallet {
 		SubscriptionDoesNotExist,
 		/// Subscription is already active
 		SubscriptionAlreadyActive,
-		/// Subscription is already paused
-		SubscriptionAlreadyPaused,
+		/// Subscription's state transition is not possible
+		SubStateTransitionImpossible,
```

Comment:
@juangirini small comment, feel free to ignore. A possible better name could be SubscriptionInvalidTransition.